crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -378,33 +380,32 @@ async fn sys_validation_workflow_inner(
             warrants.push(warrant_op);
         }
 
-        let warrant_op_hashes = warrants
-            .iter()
-            .map(|w| (w.as_hash().clone(), w.dht_basis()))
-            .collect::<Vec<_>>();
-
-        workspace
+        let authored_warrants = warrants.clone();
+        let warranted = workspace
             .authored_db
             .write_async(move |txn| {
-                for warrant_op in warrants {
+                let mut warranted = 0;
+                for warrant_op in authored_warrants {
                     insert_op_authored(txn, &warrant_op)?;
-                    summary.warranted += 1;
```

Comment:
This never counted up the summary's number of warranted ops, because it was copied in the closure.

Reply:
Oh dear, easily done and that's what tests are for :(